<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated with glade -->
<!-- Edited with love -->
<interface>
  <requires lib="gtk+" version="3.14"/>
  <template class="HyScanGtkRecorder" parent="GtkGrid">
    <property name="visible">True</property>
    <property name="can_focus">False</property>
    <property name="valign">start</property>
    <property name="row_spacing">4</property>
    <property name="column_spacing">4</property>
    <property name="column_homogeneous">True</property>
    <child>
      <object class="GtkLabel" id="track_type_label">
        <property name="visible">True</property>
        <property name="can_focus">False</property>
        <property name="halign">end</property>
        <property name="label" translatable="yes">Track type</property>
      </object>
      <packing>
        <property name="left_attach">0</property>
        <property name="top_attach">0</property>
      </packing>
    </child>
    <child>
      <object class="GtkLabel" id="record_label">
        <property name="visible">True</property>
        <property name="can_focus">False</property>
        <property name="halign">end</property>
        <property name="label" translatable="yes">Record</property>
      </object>
      <packing>
        <property name="left_attach">0</property>
        <property name="top_attach">1</property>
      </packing>
    </child>
    <child>
      <object class="GtkComboBoxText" id="track_type_cbt">
        <property name="visible">True</property>
        <property name="can_focus">False</property>
        <property name="active">0</property>
        <items>
          <item id="survey" translatable="yes">Survey</item>
          <item id="tack" translatable="yes">Tack</item>
          <item id="calibration" translatable="yes">Calibration</item>
        </items>
        <signal name="changed" handler="track_type_changed" swapped="yes"/>
        <signal name="scroll-event" handler="track_type_scrolled" swapped="yes"/>
      </object>
      <packing>
        <property name="left_attach">1</property>
        <property name="top_attach">0</property>
        <property name="width">2</property>
      </packing>
    </child>
    <child>
      <object class="GtkGrid" id="record_grid">
        <property name="visible">True</property>
        <property name="can_focus">False</property>
        <property name="column_spacing">4</property>
        <child>
          <object class="GtkSwitch" id="record_switch">
            <property name="visible">True</property>
            <property name="can_focus">True</property>
            <property name="hexpand">True</property>
            <signal name="state-set" handler="record_switched" swapped="yes"/>
          </object>
          <packing>
            <property name="left_attach">0</property>
            <property name="top_attach">0</property>
          </packing>
        </child>
        <child>
          <object class="GtkButton" id="new_track_btn">
            <property name="width_request">19</property>
            <property name="visible">True</property>
            <property name="can_focus">True</property>
            <property name="receives_default">True</property>
            <signal name="clicked" handler="new_track_clicked" swapped="yes"/>
            <child>
              <object class="GtkLabel" id="new_track_label">
                <property name="visible">True</property>
                <property name="can_focus">False</property>
                <property name="halign">center</property>
                <property name="valign">center</property>
                <property name="label" translatable="yes">New</property>
              </object>
            </child>
          </object>
          <packing>
            <property name="left_attach">1</property>
            <property name="top_attach">0</property>
          </packing>
        </child>
      </object>
      <packing>
        <property name="left_attach">1</property>
        <property name="top_attach">1</property>
        <property name="width">2</property>
      </packing>
    </child>
  </template>
  <object class="GtkSizeGroup" id="rec_ctrl_sizegroup">
    <property name="mode">vertical</property>
    <widgets>
      <widget name="track_type_cbt"/>
      <widget name="record_switch"/>
      <widget name="new_track_btn"/>
    </widgets>
  </object>
</interface>
